facedancer/device.py

```diff
@@ -230,8 +230,6 @@ def emulate(self, *coroutines: Iterable[Coroutine]):
 
         try:
             self.run_with(*coroutines)
```

Comment:
I suspect the original reasoning for this was to allow keyboard emulations to forward user-inputted ctrl+c to the target but I agree this would be better handled in user code.